doc/api/player_options.md

```diff
@@ -586,8 +586,11 @@ fullscreen, specific environments), you might not want to activate this limit.
 
 --
 
-:warning: This option will have no effect for contents loaded in _DirectFile_
-mode (see [loadVideo options](./loadVideo_options.md#prop-transport)).
+:warning: This option will have no effect for contents loaded :
+- In _DirectFile_ mode (see [loadVideo options]
+(./loadVideo_options.md#prop-transport)).
+- On Firefox browsers (version > 71) because of the lack of trust concerning
+the current video element size.
```

Comment:
Maybe we should add something about PIP in there (like "in the (unknown) event Picture-In-Picture mode is enabled").